src/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { Component } from "@angular/core";
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormsModule } from "@angular/forms";
+import { By } from "@angular/platform-browser";
+import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from "@angular/platform-browser-dynamic/testing";
+
+import { BreadcrumbComponent } from "./breadcrumb.component";
+import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
+import { BreadcrumbItem } from "./breadcrumb-item.interface";
+import { OverflowMenu } from "../dialog/overflow-menu/overflow-menu.component";
+
+@Component({
+	selector: "test-breadcrumb",
+	template: `
+	<ibm-breadcrumb [noTrailingSlash]="_noTrailingSlash">
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 1
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 2
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 3
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 4
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 5
+		</ibm-breadcrumb-item>
+	</ibm-breadcrumb>`,
+	entryComponents: [BreadcrumbComponent]
+})
+class TestBreadcrumbComponent {
+	_noTrailingSlash = true;
+
+	set noTrailingSlash(val: boolean) {
```

Comment:
AFAIK this is the recommended pattern for testing `@Input()` properties, see https://medium.com/@AikoPath/testing-angular-components-with-input-3bd6c07cfaf6

Reply:
I still see no benefit of this :\ There is already a component that passes its properties to breadcrumb's inputs so that should cover any angular change detection issues.

Do tests fail if you set it directly?